### PR TITLE
Fixed #26104 -- Fixed TypeError when passing number to forms.DurationField

### PR DESCRIPTION
**Pull request by sasha0**

### CODE REVIEW

**Comment by charettes** [comment]
Unrelated test failure.

**Comment by sasha0** [comment]
buildbot, test this please

**Comment by sasha0** [comment]
@charettes it's fine now.

File: tests/forms_tests/tests/test_fields.py [hunk 1]
```diff
@@ -722,6 +722,10 @@ class DurationForm(Form):
             str(f['duration'])
         )
 
+    def test_durationfield_3(self):
```
Comment (timgraham) [inline tests/forms_tests/tests/test_fields.py]: I'd suggest to use a more descriptive test name, otherwise looks okay.

Comment (sasha0) [inline tests/forms_tests/tests/test_fields.py]: Maybe I should add a note to release notes also?

Comment (timgraham) [inline tests/forms_tests/tests/test_fields.py]: I don't think it's much of a "feature" (if you meant for 1.10) and I didn't see any justification for a backport (if you meant 1.9.2).

Comment (sasha0) [inline tests/forms_tests/tests/test_fields.py]: Ok, renamed test.

**Comment by timgraham** [comment]
merged in 956cde800490d2d9e1e5fcc83755a9226de27f24, thanks!

**Pull request closed without merging**